### DISCUSSION TITLE
You now have write access to serverless-webpack

### DISCUSSION THREAD

**Discussion by HyperBrain** [serverless-heaven/serverless-webpack#1536] [Team Posts]
Hi @serverless-heaven/serverless-webpack-team ,

you now have write access to serverless-webpack. Hopefully you can help me a bit in maintaining the project properly, as I have not had really much time recently.
However, we should continue the project in the known quality, i.e. follow strict SEMVER and create milestones for the upcoming versions, *tag the possible PR's and issues accordingly and merge only, if all tests pass*. That also means that committers have to add unit tests for their additions accordingly.
There are many companies and people relying on the plugin (currently we have 12k downloads per day 😄  ) 

I checked some of the PRs and issues right now. We should target 5.3.0 as next release and include non-breaking feature and bugfix PRs that are agreed upon by the attached discussions.

Nevertheless I will try to get some more free time for the project again and support you as much as possible (e.g. releasing to NPM).

**Comment by hassankhan** [comment]
Thanks @HyperBrain for the effort you've put into keeping this project healthy, it has been a huge help to all of us, I'm sure.

Would it perhaps be an idea to create a `CONTRIBUTING.md` file in the repo detailing these things not just for us, but for potential future contributors as well?

**Comment by HyperBrain** [comment]
> create a CONTRIBUTING.md

Great idea. I will try to work on something the next weekend.
